### ISSUE TITLE
Inefficient use of space

### ISSUE THREAD

**Issue by ayyghost** [Cryptodog/cryptodog#2]
I feel that Cryptodog should have the same general look-and-feel of the current web client and/or the original Cryptocat. We could make a few changes here and there, but the overall style has worked well in the past, and there's no immediate reason to go a *completely* different direction.

However, I'm thinking we could make use of space more efficiently. Consider the current method of verifying someone:

![verify](https://cloud.githubusercontent.com/assets/13933477/12080248/f16bb356-b222-11e5-86de-635285c8e9f3.png)

In order to fit the verification bubble within the bounds of the app window, the Cryptocat developers decided to cover up the majority of chat messages. While certainly tolerable for a short while, this is a UI disaster. We could fix it by utilizing the tons of empty space we have on either side of the window, creating room for other things such as displaying user info and uploading files.

A design for this would have to be sketched out to maximize UX.

**Comment by MullacGeek** [comment]
How do we handle cases when the browser window is the same size as the chat box?
![Cryptodog window](https://cloud.githubusercontent.com/assets/7097800/12080432/3506cc44-b21f-11e5-98c2-73c7a87065db.png)

**Comment by ayyghost** [comment]
By expanding the chat box on the sides. Those new areas become part of the app, and when the browser is resized smaller than the app, a horizontal scrollbar appears.

**Comment by MullacGeek** [comment]
That isn't really good UX, some browsers have really subtle scrollbars.

That screenshot I posted has a horizontal scrollbar in it.

**Comment by ayyghost** [comment]
Right, we can't leave those new areas blank if we want to do this. They have to be occupied by something at all times, so the user knows it's part of the app.

**Comment by MullacGeek** [comment]
Maybe we could do something with that unused header space?

Maybe have the dialog be minimize itself to header when you click out of the box?

**Comment by ayyghost** [comment]
Possibly, but I don't like the idea of chat being covered up at any time. It's a hassle because you have to click back and forth between ongoing chat and the result of the verification attempt.

**Comment by MullacGeek** [comment]
Maybe the question-asking could be changed into a special type of message in PM with the person you're trying to verify.

**Comment by thecrazyeel** [comment]
the only thing I have to say is that NADIM is not only hot but he is a master hacker, you both smartasses!

**Comment by ayyghost** [comment]
@MullacGeek That's a significantly better idea.
